app/generators/cinema_halls/generate_seats.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module CinemaHall
+  module Generators
+    class GenerateSeats
```

Comment:
The preferred convention is to name classes with nouns (as they represent things), and methods with verbs (as they represent actions). There are exceptions of course (e.g. getters methods in ruby are just nouns) but in most cases, it's good to follow that convention.

So in this case it would be e.g. `SeatsGenerator`.